projects/storefrontapp-e2e-cypress/cypress/helpers/my-company/my-company-form.ts

```diff
@@ -0,0 +1,130 @@
+import { MyCompanyConfig, MyCompanyRowConfig } from './models/index';
+import { loginAsMyCompanyAdmin, scanTablePagesForText } from './my-company';
```

Comment:
Maybe `my-company.ts` should be renamed to something like `utils..`? It seems that this file consists of utils-like functions